app/controllers/hello_world_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HelloWorldController < ApplicationController
+  def index
+    @hello_world_props = { name: "Stranger" }
```

Comment:
We setup some simple props to show how you pass props from the controller to the view's react component, which is rendered with the react_on_rails view helper `render_component`.